.travis.yml

```diff
@@ -1,13 +1,15 @@
 ---
+sudo: false
 language: ruby
 
+notifications:
+  email: false
+
 rvm:
   - 2.1.9
   - 2.3.1
 
-sudo: false
-
-script: 'bundle exec rake validate rubocop'
-
-notifications:
-  email: false
+before_install: gem install bundler -v 1.15.3
```

Comment:
Good to know, it was part of the skeleton generated by `bundle gem rzo`.  I'll remove it.